admin/class-acc_user_importer-admin.php

```diff
@@ -277,77 +277,22 @@ private function proccess_user_data ( $users ) {
 			return $api_response;
 		}
 		
-		//remove record entirely if membership number is missing
-		$malformed_users = [];
-		foreach ( $users as $key => $user ) {
-			$member_id = $user['MEMBERSHIP_N'];
-			if (! (is_numeric($member_id) )) {
-				$malformed_users[] = $user;
-				unset( $users[$key] );
-				continue;
-			}
-		}
```

Comment:
Can you explain why you are removing the above code?  Just want to make sure I understand.  If I get it right, it is because we don't care about MEMBERSHIP_N anywhere down in the code, is that right?  I mean, we do not store it in the local database.

Reply:
The check is still there, I just moved into the main member loop, see [here](https://github.com/acc-wp/acc_user_importer/pull/2/files#diff-531c7366e93ff0ef6b25e7bec8fa16bd824918fcae7149891322397418556205R314).